templates/web/base/waste/bulky/confirmation.html

```diff
@@ -15,7 +15,13 @@ <h1 class="govuk-panel__title">
   </h1>
   <div class="govuk-panel__body">
       <p>Our contractor will collect the items you have requested on [% cobrand.bulky_nice_collection_date(report) %].</p>
-      <p>Item collection starts from [% cobrand.bulky_nice_collection_time %]. Please have your items ready and dismantled if required.</p>
+      <p>Item collection starts from [% cobrand.bulky_nice_collection_time %].&nbsp;
```

Comment:
No need to change, I just wondered why it was switched to a nbsp?

Reply:
I thought the removal of spaces with ```[%~``` would mean there was no space after the full stop

Reply:
Yeah, but it's HTML, you didn't have to use `[%~` :)